client/js/middleware/qbank.js

```diff
@@ -74,6 +74,25 @@ function getBody(userInput, question){
   }
 }
 
+/**
+ * Sends a post request to qbank with qbank specific fields already set.
+ */
+function postQbank(state, url, body = {}, headers = {}, params={}){
```

Comment:
This was getting duplicated in every api call we were making, so I extracted it into a wrapper method